functional-tests/fn_bridge_reassignment.py

```diff
@@ -0,0 +1,106 @@
+from utils import get_bridge_pubkey, wait_until, wait_until_with_value
+
+# Local constants
+# D BTC
+DEPOSIT_AMOUNT = DEFAULT_ROLLUP_PARAMS["deposit_amount"]
+# BTC Operator's fee for withdrawal
+OPERATOR_FEE = DEFAULT_ROLLUP_PARAMS["operator_fee"]
+# BTC extra fee for withdrawal
+WITHDRAWAL_EXTRA_FEE = DEFAULT_ROLLUP_PARAMS["withdraw_extra_fee"]
+# Dispatch assignment duration
+DISPATCH_ASSIGNMENT_DURATION = DEFAULT_ROLLUP_PARAMS["dispatch_assignment_dur"]
+
+
+@flexitest.register
+class BridgeWithdrawReassignmentTest(testenv.BridgeTestBase):
+    """
+    Makes two DRT deposits, then triggers the withdrawal.
+    The bridge client associated with assigned operator id is stopped.
+    After the dispatch assignment duration is over,
+    Check if new operator is being assigned or not
+    Ensure that the withdrawal resumes and completes successfully
+    """
+
+    def __init__(self, ctx: flexitest.InitContext):
+        ctx.set_env(testenv.BasicEnvConfig(101, n_operators=3, pre_fund_addrs=True))
+
+    def main(self, ctx: flexitest.RunContext):
+        address = ctx.env.gen_ext_btc_address()
+        withdraw_address = ctx.env.gen_ext_btc_address()
+        el_address = self.eth_account.address
+        self.debug(f"Address: {address}")
+        self.debug(f"Change Address: {withdraw_address}")
+        self.debug(f"EL Address: {el_address}")
+
+        btc_url = self.btcrpc.base_url
+        btc_user = self.btc.props["rpc_user"]
+        btc_password = self.btc.props["rpc_password"]
+        bridge_pk = get_bridge_pubkey(self.seqrpc)
+        self.debug(f"Bridge pubkey: {bridge_pk}")
+
+        original_balance = get_balance(withdraw_address, btc_url, btc_user, btc_password)
+        self.debug(f"BTC balance before withdraw: {original_balance}")
+
+        # Check initial balance is 0
+        balance = int(self.rethrpc.eth_getBalance(el_address), 16)
+        assert balance == 0, "Strata balance is not expected (should be zero initially)"
+
+        # Perform two deposits
+        self.deposit(ctx, el_address, bridge_pk)
+        self.deposit(ctx, el_address, bridge_pk)
+
+        # withdraw
+        self.withdraw(el_address, withdraw_address)
+
+        new_balance = get_balance(withdraw_address, btc_url, btc_user, btc_password)
+        self.debug(f"BTC balance after withdraw: {new_balance}")
+
+        # Check assigned operator
+        duties = self.seqrpc.strata_getBridgeDuties(0, 0)["duties"]
+        withdraw_duty = [d for d in duties if d["type"] == "FulfillWithdrawal"][0]
+        assigned_op_idx = withdraw_duty["payload"]["assigned_operator_idx"]
+        assigned_operator = ctx.get_service(f"bridge.{assigned_op_idx}")
+        self.debug(f"Assigned operator index: {assigned_op_idx}")
+
+        # Stop assigned operator
+        self.debug("Stopping assigned operator ...")
+        assigned_operator.stop()
+
+        # Let enough blocks pass so the assignment times out
+        # (64 is the dispatch_assignment_duration)
```

Comment:
This comment is no longer necessary, I think because `64` has been moved to a constant.